src/content/models.py

```diff
@@ -164,6 +164,12 @@ class Meta:
         null=True,
     )
 
+    confirmation_page_needed_at = models.DateTimeField(
```

Comment:
I feel this could be clearer, maybe something like
```suggestion
    confirmed_needed_at = models.DateTimeField(
```
... IIRC it's tracking the most recent datetime that someone said "yes we do still need this page"